versions.yaml

```diff
@@ -163,6 +168,10 @@ assets:
       url: "https://github.com/confidential-containers-demo/\
       linux/archive/refs/tags/"
       tag: "efi-secret-v5.17-rc6"
+    snp:
+      description: "Linux kernel that supports AMD SEV-SNP for VMs (not host kernel)"
+      url: "https://github.com/torvalds/linux/archive/refs/tags"
+      tag: "v5.19"
```

Comment:
See #4861 - @ryansavino replaced `tag:` with `version:` (for the regular SEV kernel) and did some handling for that. If we end up rebasing this PR on top of #4861 (when it's merged), then you want to use the same guest kernel as the plain SEV (5.19 includes both modules: efi_secret (needed for SEV) and sev_guest (needed for SNP) ).

Reply:
This patch could be dropped entirely now that @ryansavino's patch got merged.

Reply:
I did indeed update it to `version:` so that it would build successfully using the script. And if I understood you correctly @dubek: you are contemplating if the same kernel (config) can be used for SEV and SNP? As you mentioned, SEV and SNP have different components that are built into the kernel right now. I think this might be a longer discussion though (that might best be moved elsewhere?)

Reply:
Incidentally, 5.19 includes both efi_secret (used for SEV) and sev-guest (used for SNP) kernel modules. So at the moment, the same kernel can be used for both.

I still think it's OK to have two separate config entries for `sev` and `snp`, which currently both point to the same kernel version.